src/consensus/pbft/libbyz/Commit.cpp

```diff
@@ -34,7 +34,7 @@ void Commit::re_authenticate(Principal* p)
 
 bool Commit::verify()
 {
-  return verified_auth;
+  return true;
```

Comment:
Perhaps add a comment here and link to a GitHub ticket?

Reply:
This will be removed in the next PR